### ISSUE TITLE
[Quest][Npc] The Upper World Pillar Fragment - Abbyssion resetting too early - Deepholm

### ISSUE THREAD

**Issue by Priestelar** [WoWManiaUK/Blackwing-Lair#4082]
https://www.wowhead.com/quest=26876/the-world-pillar-fragment
https://www.wowhead.com/npc=44289/abyssion

1 - Evading mob - Really struggled on this quest, Abbyssion would often "evade" or reset after a certain amount of time and then fly away and despawn making the quest not doable unless you hardcore try to burst the mob down. Took me and my friend a bunch of tries to try and beat this mob because it would reset and desapawn.

2 - Does not drop the quest item for more than one 1 player but it should be lootable by everyone eligable in the party.

**Comment by Rushor** [comment]
also: hoverstate remove missing

**Comment by Druhl** [comment]
Confirmed.

I also experienced the early reset bug. I was mostly just running around avoiding the attacks and then Abbyssion just flew away, after 20-30 seconds. I did this a few times!

When I attacked him in earnest, I got him before reset and collected quest reward as normal.

Also: I think the friendly Earthen Ring NPCs present are supposed to also attack Abbyssion but they just stand there (not sure).
